CODEOWNERS

```diff
@@ -76,8 +76,8 @@ go.mod @fleetdm/go
 #
 # (see website/config/custom.js for DRIs of other paths not listed here)
 ##############################################################################################
-/docs                                           @rachaelshaw
-/docs/Using-Fleet/REST-API.md                   @rachaelshaw # « REST API reference documentation
+/docs                                           @rachaelshaw @eashaw @noahtalerman
+/docs/REST\ API/rest-api.md                   @rachaelshaw @lukeheath # « REST API reference documentation
```

Comment:
1 DRI not 2

Reply:
@mikermcneil This is to provide backups while @rachaelshaw is out.  Are you saying we should remove her and just list me?